config/default.yml

```diff
@@ -1677,6 +1677,12 @@ Rails/Exit:
   Exclude:
     - lib/**/*.rake
 
+Rails/FilePath:
+  EnforcedStyle: slashes
```

Comment:
Not sure we should change the default, as this would break the CI for everyone using this cop.